homeassistant/components/speedtestdotnet/__init__.py

```diff
@@ -46,8 +48,8 @@
                     ): cv.positive_time_period,
                     vol.Optional(CONF_MANUAL, default=False): cv.boolean,
                     vol.Optional(
-                        CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)
-                    ): vol.All(cv.ensure_list, [vol.In(list(SENSOR_TYPES))]),
+                        CONF_MONITORED_CONDITIONS, default=list(SENSOR_KEYS)
```

Comment:
The YAML configuration is deprecated for this integration. We should remove the import and schema instead.

Reply:
PR #55642 submitted to remove config import.

Reply:
This was changed to 2021.9, and will cause issues for those that still have it in their configuration, I think this PR should be included in a hotfix 2021.9, and the removal PR which will be a breaking change in 2021.10

Reply:
Ok, I guess that's best.